docs/codeflow/integrating-web-publisher.md

```diff
@@ -111,8 +111,8 @@ To help your users easily find their way to Web Publisher on your site or repo,
 
 | Button preview | Direct URL |
 | --- | --- |
-| <img alt="Edit in Web Publisher" src="./assets/button-edit-in-web-publisher-icon-medium.svg" /> | <a href="./assets/button-edit-in-web-publisher-icon-medium.svg" target="_blank">button-edit-in-web-publisher-icon-medium.svg</a> |
-| <img alt="Edit in Web Publisher" src="./assets/button-edit-in-web-publisher-icon-small.svg" /> | <a href="./assets/button-edit-in-web-publisher-icon-small.svg" target="_blank">button-edit-in-web-publisher-icon-small.svg</a> |
+| <img alt="Edit in Web Publisher" src="/img/button-edit-in-web-publisher-icon-medium.svg" /> | <a href="/img/button-edit-in-web-publisher-icon-medium.svg" target="_blank">button-edit-in-web-publisher-icon-medium.svg</a> |
+| <img alt="Edit in Web Publisher" src="/img/button-edit-in-web-publisher-icon-small.svg" /> | <a href="/img/button-edit-in-web-publisher-icon-small.svg" target="_blank">button-edit-in-web-publisher-icon-small.svg</a> |
```

Comment:
So applying the same naming convention would give us:

```
edit_in_web_publisher.svg
edit_in_web_publisher_small.svg
```

(I'd avoid the `_medium` suffix for the default size. See the `open_in_stackblitz.svg` image for precedent.)

Reply:
This makes sense. Thank you!